spec/features/course/assessment/answer/text_response_answer_spec.rb

```diff
@@ -40,13 +41,27 @@
 
           # The file should show in the dropzone
           expect(dropzone).to have_css('span', text: 'text.txt')
-
-          click_button('Save Draft')
+          wait_for_job
 
           expect(dropzone).to have_no_css('span', text: 'text.txt')
           expect(file_view).to have_css('span', text: 'text.txt')
           expect(file_view).to have_css('span', count: 2)
           expect(answer.specific.attachments).not_to be_empty
+
+          # Attempting to delete the file that we have just uploaded
+          # Click on the delete button and confirm
+          delete_button = file_view.find('svg')
```

Comment:
What if there's any other svg element? This test will fail unexpectedly

Reply:
this is guaranteed here that only one file will be existing within the `file_view` (since we only upload 1 file named `text.txt` into the text response answer here). If apparently there are more than 1 files, means the test fails before it reaches here, and hence it's quite safe to do this operation for finding the `delete_button`.

Reply:
What if I add another svg element that's not of file upload? The point here is that the test should not break when some unrelated code changes occur!

Reply:
understand on this.

Done